pkg/config/config.go

```diff
@@ -248,26 +239,30 @@ func (c *ConfigImpl) UseContext(ctx context.Context, name string) error {
 	return c.Save()
 }
 
-func (c *ConfigImpl) CreateContext(ctx context.Context, name, token, url string) error {
+func (c *ConfigImpl) CreateContext(ctx context.Context, name, token, url, caCert string) error {
 	if _, exists := c.Contexts[name]; exists {
 		return fmt.Errorf("authentication context with the name \"%s\" already exists", name)
 	}
 
 	authCtx := &AuthContext{
-		Name:  name,
-		URL:   url,
-		Token: token,
-		Type:  "APIKey",
-		Beta:  false,
```

Comment:
`Beta` was always `false` (also in codefresh cli v1, if i am not mistaked), and was also never actually used for anything.